pkg/types/v1/logger.go

```diff
@@ -33,7 +29,6 @@ type Logger interface {
 	Debug(...interface{})
 	Error(...interface{})
 	Fatal(...interface{})
-	Success(...interface{})
```

Comment:
Why does the interface need to change? shouldn't whatever other tool we use just follow the interface?

Reply:
The other tool that used this was luet and changing that was much more difficult than adapting our logger to confirm to let standards. Otherwise we got the default let logger which logged very differently from 3lemental and produced a very ugly output between both apps.

Now that we don't use luet in here we can safely drop our logger interface to the minimum needed as we are the only ones using it. 

Those methods were only used by luet, anything under elemental was using the simple debug,info,fatal,warn facilities

Reply:
Mainly to avoid the dissonance between both loggers, we adapted ours and passed it to luet so we could manage it's output properly, but let needed this interfaces methods.

Reply:
gotcha